cmd/create.go

```diff
@@ -110,19 +109,20 @@ func newCreateConfig(args []string) createConfig {
 // Skipped if not in an interactive terminal (non-TTY), or if --confirm (agree to
 // all prompts) was not explicitly set.
 func (c createConfig) Prompt() createConfig {
-	name := deriveName(c.Name, c.initConfig.Path)
 	if !interactiveTerminal() || !c.initConfig.Confirm {
 		// Just print the basics if not confirming
 		fmt.Printf("Project path: %v\n", c.initConfig.Path)
-		fmt.Printf("Project name: %v\n", name)
+		fmt.Printf("Project name: %v\n", c.initConfig.Name)
 		fmt.Printf("Runtime: %v\n", c.Runtime)
 		fmt.Printf("Trigger: %v\n", c.Trigger)
 		return c
 	}
+
+	derivedName, derivedPath := deriveNameAndPath(prompt.ForString("Project name", "", prompt.WithRequired(true)))
 	return createConfig{
 		initConfig: initConfig{
-			Path:    prompt.ForString("Project path", c.initConfig.Path),
-			Name:    prompt.ForString("Project name", name, prompt.WithRequired(true)),
+			Name:    derivedName,
+			Path:    derivedPath,
```

Comment:
It appears to me that perhaps this omits the prompting of name and path entirely, right?

I was actuall fond of the original idea:  to prompt if the parameter were not provided as a flag.  If that's not the plan anymore, no problem!  But if we wanted to prompt the user when the parameters had not been provided, would we not instead leave things roughly as they were, and insteaed add here a conditional that only prompts if the flag does not exist (as provided by the CLI API)?

Reply:
> It appears to me that perhaps this omits the prompting of name and path entirely, right?

@lkingland I may be misreading your comment, so disregard if so (or correct me). The prompt occurs on line 121.

```js
derivedName, derivedPath := deriveNameAndAbsolutePathFromPath(prompt.ForString("Project path", c.initConfig.Path, prompt.WithRequired(true)))
```